src/components/button/Button.tsx

```diff
@@ -1,43 +1,45 @@
-// @flow
 import * as React from 'react';
 import classNames from 'classnames';
-import omit from 'lodash/omit';
-
 import LoadingIndicator from '../loading-indicator';
 import RadarAnimation from '../radar';
 
-type Props = {
+export enum ButtonType {
```

Comment:
😇

Reply:
surprising there isn't a TS global for this